src/controllers/update/update.use.paper.controller.ts

```diff
@@ -8,6 +8,9 @@ export const get = (req: Request, res: Response) => {
   return res.render(config.USE_PAPER_PAGE, {
     backLinkUrl: config.SECURE_UPDATE_FILTER_URL,
     templateName: config.USE_PAPER_PAGE,
-    applyWithPaperFormHeading
+    applyWithPaperFormHeading,
```

Comment:
If the question is around using a boolean isUpdate instead of pageParams.isRegistration, used it as it was a pattern used by registration and also in some of the update journey